crates/metal-mir/src/types/function.rs

```diff
@@ -2,8 +2,12 @@ use super::{visibility::Visibility, Type};
 
 #[derive(Debug, Clone)]
 pub struct FunctionSignature {
+    /// Function name
     pub name: String,
+    /// Represents the type returned by this function
     pub return_type: Type,
+    /// Represents the arguments this function takes
     pub arguments: Vec<Type>,
+    /// Represents who can use this function
```

Comment:
This would fit as a description of Visibility itself, but as a description of a field this should be consistent with other cases and say something abstract like "The visibility of this function". In AST, the convention for "obvious" fields like this is to say "See \[\<link to obvious field's type\>\]", but I guess MIR skipped that train already